### PR TITLE
Code quality

### PR DESCRIPTION
**Pull request by NicholasDanks**
Not ready yet

### CODE REVIEW

File: db/migrations/003_repositories_add_quality.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Sequel.migration do
+  change do
+  	alter_table(:repositories) do
+  	  add_column :flog_score, String
+  	  add_column :flay_score, String
+      add_column :rubocop_errors, String
```
Comment (rjollet) [inline db/migrations/003_repositories_add_quality.rb:6]: it seems that the indentation is weird here. make sure you use space and not tab

**Review by rjollet — COMMENTED** [review]
can you add also some test to make sure that you got a value for flog and flay ?

File: services/load_developer_from_github.rb [hunk 2]
```diff
@@ -30,6 +30,7 @@ class LoadDeveloperFromGithub
 
     github_developer.repos.each do |gh_repo|
       write_developer_repository developer, gh_repo
+      UpdateRepository.call(gh_repo.id)
```
Comment (rjollet) [inline services/load_developer_from_github.rb]: do not call your service updaterepository can you call it score quality or quality check something more clear

File: services/add_repository_quality_data.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  register :load_repository_quality_data, lambda { |repo|
+    devname = repo.full_name.split('/').first
+    reponame = repo.full_name.split('/').last
+    quality_data = GetCloneData.new(devname, reponame)
+    if quality_data
+      repo.flog_score = quality_data.get_flog_scores
+      repo.flay_score = quality_data.get_flay_score
+      repo.rubocop_errors = quality_data.get_rubocop_errors
+
```
Comment (rjollet) [inline services/add_repository_quality_data.rb]: add repo.save()

**Comment by NicholasDanks** [comment]
Still need to:
- add tests
- filter only for Ruby
- refactor

**Comment by rjollet** [comment]
- add tests
- Change the service name UpdateRefpository shoudl have another name
- move git_clone.rb into /lib

that would be good for this pull request

**Comment by NicholasDanks** [comment]
I think the build failed because gitget hasn't been pushed to Rubygems

**Pull request merged**